src/components/tableview/TableTable.vue

```diff
@@ -27,6 +27,7 @@
           </div>
           <div class="btn-wrap">
             <button class="btn btn-primary" type="submit">Search</button>
+            <button class="btn btn-flat" type="button" @click.prevent="exportTable()">Export</button>
```

Comment:
Not happy with adding another button to this bar. It's getting more and more crowded. However, I don't really know where else it could go.

Reply:
Don't worry about that! We'll deal with it :-).

We're likely going to coalesce on some sort of `actions` button in the statusbar for export / adding rows / viewing table schema information, etc, so it will go there, but we need to experiment with it some more.